_scalingo_api/events.md

```diff
@@ -1211,11 +1211,19 @@ _When:_ Each time an SCM integration link is created to an app
 `type=link_scm`
 
 {:.table}
-| field           | type   | description                                 |
-| --------------- | ------ | ------------------------------------------- |
-| repo_name       | string | Name of the repository                      |
-| linker_username | string | Username of the linker                      |
-| source          | string | Source URL of the repository                |
+| field                        | type    | description                                                                                          |
+| ---------------------------- | ------- | ---------------------------------------------------------------------------------------------------- |
+| repo_name                    | string  | Name of the repository                                                                               |
+| linker_username              | string  | Username of the linker                                                                               |
+| source                       | string  | Source URL of the repository                                                                         |
+| branch                       | string  | Branch on which the repository is tied                                                               |
+| auto_deploy                  | boolean | Auto-deploy the app when the parametered branches is updated                                         |
+| auto_deploy_review_apps      | boolean | Auto-deploy Review Apps when Pull/Merge Requests are opened                                          |
+| delete_on_close              | boolean | Delete the Review App when the related Pull/Merge Request is closed                                  |
+| delete_stale                 | boolean | Delete the Review App when the related Pull/Merge Request is staled                                  |
+| hours_before_delete_on_close | integer | Hours before deleting the Review Apps when the related Pull/Merge Request is closed                  |
+| hours_before_delete_stale    | integer | Hours before deleting the Review Apps when the related Pull/Merge Request is staled                  |
```

Comment:
These lines mix three ways of writing:
1. Singular (lines 1222 and 1223): "the Review App"
2. Identified plural (lines 1224 and 1225): "the Review Apps"
3. Generic plural (line 1221): "Review Apps"

They need to be aligned with one wording. I suggest to use the singular, as the action is made for one pull/merge request, and one Review App at a time:
```
| auto_deploy_review_apps      | boolean | Auto-deploy a Review App when a Pull/Merge Request is opened                                          |
| delete_on_close              | boolean | Delete the Review App when the related Pull/Merge Request is closed                                  |
| delete_stale                 | boolean | Delete the Review App when the related Pull/Merge Request is staled                                  |
| hours_before_delete_on_close | integer | Hours before deleting the Review App when the related Pull/Merge Request is closed                  |
| hours_before_delete_stale    | integer | Hours before deleting the Review App when the related Pull/Merge Request is staled                  |
```

Reply:
Right once again, fixing this